### ISSUE TITLE
D_Wipe bug fix (?)

### ISSUE THREAD

**Issue by kraflab** [coelckers/prboom-plus#236]
I made a change to the D_Wipe method in dsda-doom and maybe it is of interest to mainline pr+.

There is a timing "bug" with the wipe that causes the game to skip frames based on...I'm not sure what. People with different hardware get different results. Basically the inputs will look like this:

```
frame 0 -> get input
(wipe happens)
skip to frame x -> get input
```

The "catch up" mechanism here thinks the game is behind after the wipe: https://github.com/kraflab/dsda-doom/blob/master/prboom2/src/d_client.c#L340

Then it skips forward 7 frames (or fewer) until the break condition here: https://github.com/kraflab/dsda-doom/blob/master/prboom2/src/d_client.c#L348

This is a pretty big problem for speedrunning, since you lose frames, and especially because it doesn't seem to happen to everyone. So I fixed it by leveraging a safety mechanism that was evidently introduced to fix another problem, here: https://github.com/kraflab/dsda-doom/commit/642da66cd854358aafb61644074196c8519ae365

Basically the `force_singletics_to` variable means "don't skip ahead to catch up".

I'm not sure if this is relevant for pr+ or if it is "supposed to" do this... 😄

**Comment by kraflab** [comment]
Cleaning up (closing) some old issues
